proof/refine/ARM/Finalise_R.thy

```diff
@@ -2326,6 +2326,295 @@ lemma replyClear_invs'[wp]:
   apply wpsimp
   sorry
```

Comment:
Will this be the last sorry in Finalise_R?

Reply:
There is also `replyPop_iflive` and `replyRemove_corres` left

Reply:
Michael took care of the first and the latter is a temporary sorry for what Corey is currently working on in IpcCancel_R.